docs/_adapters/cpx.md

```diff
@@ -38,12 +38,12 @@ The {{page.name}} can install **{{page.version}}** of the {{page.mesh_name}}.
 
 The {{ page.name }} includes a handful of sample applications. Some of these applications are from other service meshes and some of these sample applications are general-purpose examples. Use Meshery to deploy any of these sample applications.
 
-- [Bookinfo]({{ site.baseurl }}/guides/deploying-sample-apps#bookinfo)
+- [Bookinfo]({{ site.baseurl }}/guides/sample-apps#bookinfo)
     - Follow this [tutorial workshop](https://github.com/layer5io/istio-service-mesh-workshop/blob/master/lab-2/README.md) to set up and deploy the BookInfo sample app on Istio using Meshery. 
-- [Httpbin]({{ site.baseurl }}/guides/deploying-sample-apps#httpbin)
+- [Httpbin]({{ site.baseurl }}/guides/sample-apps#httpbin)
     - Httpbin is a simple HTTP request and response service.
-- [Hipster]({{ site.baseurl }}/guides/deploying-sample-apps#hipster)
-    - Hipster Shop Application is a web-based, e-commerce demo application from the Google Cloud Platform.
+- [Online Boutique]({{ site.baseurl }}/guides/sample-apps#onlinebouique)
```

Comment:
```suggestion
- [Online Boutique]({{ site.baseurl }}/guides/sample-apps#online-boutique)
```
also look at other places as well.

Reply:
Thanks, @leecalcote  @afzal442  Are you saying any place there are two words there should be (-) at the URL?

Reply:
Thanks, @leecalcote  @afzal442  Are you saying any place there are two words there should be (-) at the URL?
 Like Image Hub, Online Boutique, Linkered Books? Because I wrote Image Hub URL together and you said it's fine.  [Image Hub]({{ site.baseurl }}/guides/sample-apps/imagehub)

Reply:
Thanks, @leecalcote  @afzal442  Are you saying any place there are two words there should be (-) at the URL?
 Like Image Hub, Online Boutique, Linkered Books? Because I wrote Image Hub URL together and you said it's fine.  [Image Hub]({{ site.baseurl }}/guides/sample-apps/imagehub)

Reply:
Thanks, @leecalcote  @afzal442  Are you saying any place there are two words there should be (-) at the URL?
 Like Image Hub, Online Boutique, Linkered Books? Because I wrote Image Hub URL together and you said it's fine.  [Image Hub]({{ site.baseurl }}/guides/sample-apps/imagehub)

Reply:
this was a workaround, not a fix; you need to see which links work; in the description of the issue itself, I have specified where those links can be found; 

>Because I wrote Image Hub URL together and you said it's fine.

see, most of the links have different tags;

further, you should check through the preview if it works or not.

Reply:
> this was a workaround, not a fix; you need to see which links work; in the description of the issue itself, I have specified where those links can be found;
> 
> > Because I wrote Image Hub URL together and you said it's fine.
> 
> see, most of the links have different tags;
> 
> further, you should check through the preview if it works or not.

Okay.